### ISSUE TITLE
Discussion on backend.

### ISSUE THREAD

**Issue by nishihere19** [purnima143/Kurakoo#8]
I would like to work on the backend of this project. Can we have a discussion on the requirements?

**Comment by himanshujaidka** [comment]
yes, we need discord server or this

**Comment by samridhikots** [comment]
I would also like to work on the backend of this project. What all are the requirements?

**Comment by namansamra** [comment]
i would like to work on the backend server. Can we have discussion?

**Comment by purnima143** [comment]
@himanshujaidka  Yes, we have a discord server for discussion regarding the project. But those who are participants of GSSOC 21 can join

**Comment by purnima143** [comment]
For backend discussion pls come to the **#kurakoo** in **GSSOC 21** server on **Discord**. Otherwise, it would create conflicts among us and we couldn't work on the project properly.

**Comment by himanshujaidka** [comment]
> @himanshujaidka Yes, we have a discord server for discussion regarding the project. But those who are participants of GSSOC 21 can join

I am a participant @purnima143

**Comment by purnima143** [comment]
@himanshujaidka That's great 😅. Just wait for a while they will give access to participants on discord

**Comment by A7abhilash** [comment]
How the authentication is needed? Is it just custom username and password or google sign in or any other method?

**Comment by himanshujaidka** [comment]
@maw1a @codewithvk @purnima143 now, pls suggest me that on which API I have to work further or the missing APIs
